app/views/articles/index.html.erb

```diff
@@ -16,7 +16,7 @@
   <meta property="og:description" content="<%= SiteConfig.community_description %>" />
   <meta property="og:site_name" content="<%= community_qualified_name %>" />
 
-  <meta name="twitter:site" content="@<%= SiteConfig.social_networks_handle %>">
+  <meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
```

Comment:
Since we make no assumptions about Twitter above, we just loop over social media types, is it assumed that Twitter will always be present? If not, consider checking before rendering Twitter related meta tags.

Same question for all Twitter related stuff below.

Reply:
@nickytonline great question. Since we using rails-settings-cached type: :hash we've set a default of nil for all the social media types so when we call SiteConfig.social_media_handles["twitter"] it will always return nil if it has not been set to a meaningful value. In addition, one of the subsequent steps for generalisation is to show the user a banner to the admin on what needs to be configured so that the site is useful. Twitter will be one of those values as we use it so often around the site.   https://github.com/thepracticaldev/dev.to/projects/15#card-36014194